code/game/objects/items/melee/baton.dm

```diff
@@ -427,6 +427,11 @@
 	on_stun_volume = 50
 	active = FALSE
 	context_living_rmb_active = "Harmful Stun"
+	light_range = 2
+	light_system = OVERLAY_LIGHT
+	light_on = FALSE
+	light_color = LIGHT_COLOR_ORANGE
```

Comment:
at default lighting power this ends up being remarkably colorless, is that what you want?

also, are you sure you want this to be an overlay light? The range makes it very oppressive, which might be nice but I think would be paired better with a weaker, more faint light (also consider a range of 1.5 for something slightly more pulled into the source).

If what you want is just for the baton to glow (and not illuminate anything else), we have an emissives system that lets you cut things out from the darkness. That might work well here depending on your goal

Reply:
I didn't want the colours to be too harsh. I'll set the range to 1.5, the goal is for it to be soft glow that can be ignored. It's meant to make things look pretty, not standout by itself.